docs/MapColonies/Raster/services/export.mdx

```diff
@@ -57,98 +52,163 @@ Our specification consists of the following:
 JSON file that contains the record metadata such as footprint and resolution
 - file extension: ".JSON"
 - provided fields follow the profile convention [here](/docs/MapColonies/Raster/services/catalog/raster-catalog-profile-v1)
-**Note:** when using create/roi API the metadata link is returned via "links" response, while in create (deprecated) API the metadata can be reached only when replacing the GPKG link suffix from ".GPKG" to ".JSON"
 
 ### Important Notes
-1. The data is returned via callback request or on /create request response (structure in appendix 1)
-2. To download the data one needs to use authentication token
+1. The data is returned via callback request or on /export-tasks request response (structure in appendix 1)
 
 ### Appendix
 
 
 <Tabs>
-<TabItem value="appendix-1" label="Callback request structure (create/roi)">
+<TabItem value="appendix-1" label="Callback request structure (export-tasks)">
 
-- status: Enum - OperationStatus.IN_PROGRESS | OperationStatus.COMPLETED | OperationStatus.FAILED
-- links: Object - consist of download links for the exported GPKG and metadata file
-- expirationTime: string($date) - date when the exported file will be deleted
-- fileSize: number - GPKG file size in bytes
-- recordCatalogId: uuid - Raster Catalog Record DB ID - from orginal request
-- description: string - the description that was sent to the create method
+- catalogRecordID: uuid - Raster Catalog Record DB ID - from orginal request
+- domain: string - the export domain ("RASTER" / "DEM" / "3D", etc.)
 - roi: FeatureCollection - requested region of intrest
-- jobId: uuid - unique identifier for export request - the field that was returned from orginal create request
+- artifactCRS: artifact crs (for Example: "4326")
+- description: string - the description that was sent to the create method
+- keywords: Object - task metadata in key-value pair
+- parameters: Object - task parameters in key-value pair
+- webhook: Object - events registered to, and to what url to call when event executed
+- id: number - unique identifier for export request
+- estimatedSize: number - estimated size of the artifacts in bytes
+- estimatedTime: number - estimated time to finish the export task
+- status: Enum - IN_PROGRESS | COMPLETED | ABORTED | PAUSED | FAILED | PENDING | EXPIRED | ARCHIVED
 - errorReason: string - if export task was unsuccessful this field will describe the error
+- progress: number - minimum: 0, maximum: 100
+- artifacts: Object - Object that describes the task products and how to access them
+- createdAt: string($date) - creation task time - UTC format
+- expiredAt: string($date) - describes when the task will be expired at - UTC format
+- finishedAt: string($date) - describes when the task reached an end state - UTC format
 
 </TabItem>
-<TabItem value="appendix-2" label="Callback Request Example (create/roi)">
+<TabItem value="appendix-2" label="Callback Request Example (export-tasks)">
```

Comment:
All examples are the same... Why is that? Are they really?

Reply:
they were copied from swagger example